packages/messages/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true
```

Comment:
Can we add `"jest": true` inside `"env"`? This will avoid getting warnings when using the global utilities from jest.